llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -1927,6 +1927,12 @@ def int_amdgcn_inverse_ballot :
   Intrinsic<[llvm_i1_ty], [llvm_anyint_ty],
             [IntrNoMem, IntrWillReturn, IntrNoCallback, IntrNoFree]>;
 
+// Lowers to S_BITREPLICATE_B64_B32.
+// The argument must be uniform; otherwise, the result is undefined.
+def int_amdgcn_s_bitreplicate :
+  Intrinsic<[llvm_i64_ty], [llvm_i32_ty],
```

Comment:
Use DefaultAttrsIntrinsic to imply all the properties defined with `IntrinsicProperty<1>` in `Intrinsics.td`.